### PR TITLE
Adds the first e2e test

### PR DESCRIPTION
**Pull request by Kuqd**
This is a sample of one e2e tests for #37 . This first test create a gameserver, waits for readiness, assert some values then finally ping it (send/receive).

The idea of this PR is to see if the direction is correct, but also I'd like to plug it to cloud builder to see if we can make it works @markmandel I'll need your help on this. ( Can we use your account to provision a cluster for e2e ? How should we proceed ?)

I would like that we list all tests for gameservers/fleets/allocation, so I can tackled them one by one. (Let's use the ticket #37 )

Locally with a GKE cluster it works.
(I'm having sometimes some timeout with the k8s api, might be from my connection
`Post https://35.185.224.168/apis/stable.agones.dev/v1alpha1/namespaces/default/gameservers: dial tcp 35.185.224.168:443: i/o timeout`)

Have a spin, I hope you like it.

### CODE REVIEW

**Comment by Kuqd** [comment]
I can confirm there is something weird with gcloud toolchain, if I use kubectl with my machine, it doesn't work anymore in the container.(MacOS)

**Comment by markmandel** [comment]
Swamped with Cloud NEXT - so will likely take a look once things are done / calmed down a bit, so I may not respond until next week.

File: build/build-image/run-individual-e2e.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cd ./build
+kubectl port-forward statefulset/consul-consul 8500:8500 &
+# waits for port-forward to setup
+sleep 5
+echo "Starting e2e tests"
+consul lock -timeout 5m -try 5m -verbose LockE2E /root/e2e.sh
+killall -q kubectl
```
Comment (Kuqd) [inline build/build-image/run-individual-e2e.sh]: I might need to add a newline here

**Comment by Kuqd** [comment]
what a ride ! I'll need to cleanup now

**Comment by Kuqd** [comment]
I'll open a new one for cleaner review !

**Pull request closed without merging**